DESCRIPTION

```diff
@@ -1,9 +1,10 @@
 Package: shinytest
 Title: Test Shiny Apps
-Version: 1.3.1.9001
+Version: 1.3.1.9002
 Authors@R: c(
     person("Winston", "Chang", role = c("aut", "cre"), email = "winston@rstudio.com"),
-    person("Gábor", "Csárdi", role = "aut", email = "gabor@rstudio.com")
+    person("Gábor", "Csárdi", role = "aut", email = "gabor@rstudio.com"),
+    person("RStudio", role = c("cph", "fnd"))
```

Comment:
Mango Solutions should also be listed in these roles, since development started there.